test-integration/operator-tests/src/test/java/org/infinispan/operator/MinimalSetupIT.java

```diff
@@ -0,0 +1,193 @@
+package org.infinispan.operator;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.function.BooleanSupplier;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.http.entity.ContentType;
+import org.assertj.core.api.Assertions;
+import org.infinispan.Caches;
+import org.infinispan.Infinispan;
+import org.infinispan.Infinispans;
+import org.infinispan.TestServer;
+import org.infinispan.identities.Credentials;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import cz.xtf.client.Http;
+import cz.xtf.core.http.Https;
+import cz.xtf.core.openshift.OpenShift;
+import cz.xtf.core.openshift.OpenShifts;
+import cz.xtf.core.waiting.SimpleWaiter;
+import cz.xtf.junit5.annotations.CleanBeforeAll;
+import io.fabric8.kubernetes.api.model.Pod;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * MinimalSetupIT
+ *
+ * Tests all the defaults as minimal infinispan cluster deployment was created.
+ * It is expected that endpoints will be unencrypted with default authentication.
+ *
+ * spec:
+ *   expose:
+ *     type: Route
+ *   replicas: 2
+ *
+ * Should result in the same as:
+ *
+ * spec:
+ *   replicas: 2
+ *   container:
+ *     cpu: 500m
+ *     memory: 512Mi
+ *   service:
+ *     type: cache
```

Comment:
Not related to code, just description side
```suggestion
 *     type: Cache
```